### PR TITLE
add auto joining parameters

### PR DESCRIPTION
**Pull request by ASoTNetworks**

### CODE REVIEW

File: examples/inventory-mainnet.yml [hunk 1]
```diff
@@ -3,9 +3,8 @@
 all:
   vars:
     ansible_user: root
-    gaiad_version: v7.0.2
     genesis_url: "https://github.com/cosmos/mainnet/raw/master/genesis.cosmoshub-4.json.gz"
-    chain_id: cosmoshub-4
+    chain_info_server: "https://rpc.cosmos.network:443"
```
Comment (YurkoWasHere) [inline examples/inventory-mainnet.yml:7]: Is this gaiad specific or do all chains have a rpc server like this?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 [![Test Gaia Versions](https://github.com/hyphacoop/cosmos-ansible/actions/workflows/test-gaia-versions.yml/badge.svg?branch=main)](https://github.com/hyphacoop/cosmos-ansible/actions/workflows/test-gaia-versions.yml)
 [![Test Gaia Versions with Stateful Genesis](https://github.com/hyphacoop/cosmos-ansible/actions/workflows/export-mainnet-status-badge.yml/badge.svg?branch=main)](https://github.com/hyphacoop/cosmos-ansible/actions/workflows/export-mainnet-status-badge.yml)
 [![Join Theta Testnet](https://github.com/hyphacoop/cosmos-ansible/actions/workflows/test-theta-node.yml/badge.svg?branch=main)](https://github.com/hyphacoop/cosmos-ansible/actions/workflows/test-theta-node.yml)
+[![Join Cosmos Mainnet](https://github.com/hyphacoop/cosmos-ansible/actions/workflows/test-mainnet-node.yml/badge.svg?branch=main)](https://github.com/hyphacoop/cosmos-ansible/actions/workflows/test-mainnet-node.yml)
```
Comment (dasanchez) [inline README.md:7]: This is amazing!
Same thing, "Join Cosmos Hub"

File: docs/Playbook-Variables.md [hunk 3]
```diff
@@ -10,11 +10,12 @@ ansible-playbook gaia.yml -i examples/inventory-local.yml --extra-vars "gaiad_ve
 | Variable                 | Description                                                                       | Example Value                                                                                       |
 |--------------------------|-----------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|
 | `target` | Target server IP/domain | `example.com` |
+| `chain_registry` | When set it will automatically define `gaiad_version`, `chain_id` and `genesis_url` variables from the chain registry file | `https://registry.ping.pub/testnets/theta/chain.json` |
```
Comment (dasanchez) [inline docs/Playbook-Variables.md:13]: Very nice

File: .github/workflows/test-mainnet-node.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  workflow_dispatch:
+  schedule:
+    # At 15:30 on Monday.
+    - cron: '30 15 * * 1'
+  pull_request:
+    types:
+      - opened
+      - closed
+jobs:
+  test-theta-testnet:
```
Comment (dasanchez) [inline .github/workflows/test-mainnet-node.yml:14]: test-join-cosmoshub

File: .github/workflows/test-mainnet-node.yml [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+
+name: Join Cosmos Mainnet
```
Comment (dasanchez) [inline .github/workflows/test-mainnet-node.yml:3]: Join Cosmos Hub

**Review by dasanchez — CHANGES_REQUESTED** [review]
This is so good, great job @ASoTNetworks!
I have one request- could you change all the file names and workflows from "mainnet" to "cosmoshub"?
- `test-mainnet-node.yml` -> `test-cosmoshub-node`
- `inventory-mainnet.yml` -> `inventory-cosmos-hub.yml`

**Pull request merged**